src/host/_stream.cpp

```diff
@@ -765,6 +765,11 @@ using Microsoft::Console::VirtualTerminal::StateMachine;
                     Status = AdjustCursorPosition(screenInfo, CursorPosition, dwFlags & WC_KEEP_CURSOR_VISIBLE, psScrollY);
                 }
             }
+            // Notify accessibility
+            if (screenInfo.HasAccessibilityEventing())
+            {
+                screenInfo.NotifyAccessibilityEventing(CursorPosition.X, CursorPosition.Y, CursorPosition.X + 1, CursorPosition.Y);

```

Comment:
> We're not really using them because they're for MSAA, it seems.

I don't quite understand that... `NotifyAccessibilityEventing` uses all 4 parameters it's given.
If you want to add new functions to `screenInfo`, just do it IMO. It's not really an API after all, since it's not an interface.

Reply:
I did you one better, just took the 2 lines of code that notify UIA and used that